src/ClientLibrary/FactoryOrchestratorClient.cs

```diff
@@ -29,13 +32,17 @@ public partial class FactoryOrchestratorClient
         /// </summary>
         /// <param name="host">IP address of the device running Factory Orchestrator Service. Use IPAddress.Loopback for local device.</param>
         /// <param name="port">Port to use. Factory Orchestrator Service defaults to 45684.</param>
-        public FactoryOrchestratorClient(IPAddress host, int port = 45684)
+        /// <param name="serverIdentity">Distinguished name for the server.</param>
+        /// <param name="certhash">Hash value for the server certificate.</param>
+        public FactoryOrchestratorClient(IPAddress host, int port = 45684, string serverIdentity = "FactoryServer", string certhash = "E8BF0011168803E6F4AF15C9AFE8C9C12F368C8F")

```

Comment:
What do we think about allowing a user to specify a (optional) custom callback, @patelp3989 ? Can you think of any downsides?